app/src/main/java/com/locoquest/app/RetrofitClientInstance.kt

```diff
@@ -6,14 +6,14 @@ import retrofit2.converter.moshi.MoshiConverterFactory
 
 object RetrofitClientInstance {
     private var retrofit: Retrofit? = null;
-    private val BASE_URL = "https://geodesy.noaa.gov/api/nde/"
+    private const val BASE_URL = "https://geodesy.noaa.gov/api/nde/"
```

Comment:
Added const to variable, it does not change
https://kotlinlang.org/docs/null-safety.html#safe-calls